src/Components/Web.JS/src/Boot.Server.ts

```diff
@@ -27,6 +28,8 @@ async function boot(userOptions?: Partial<CircuitStartOptions>): Promise<void> {
   const options = resolveOptions(userOptions);
   const logger = new ConsoleLogger(options.logLevel);
   window['Blazor'].defaultReconnectionHandler = new DefaultReconnectionHandler(logger);
+  window['Blazor']._internal.InputFile = InputFile;
```

Comment:
Rather than patching `window['Blazor']._internal` like this, would it work to do the same for `GlobalExports` as you did for `InputFile`? That is, have a peer-level `WasmGlobalExports.ts` that extends `GlobalExports`.

I don't mind too much, but it would be easier to make sense of what is being exported if you can see it statically from the `import` declarations at the top instead of having to read all the code to see things get added dynamically.

Reply:
Let me try this as a separate follow. I might need your help because I'm not super sure how to express this in an idiomatic way.